app/view_models/workarea/storefront/search_autocomplete_view_model.rb

```diff
@@ -21,8 +27,16 @@ def searches
         )
       end
 
-      def no_results?
-        products.empty? && searches.empty?
+      def trending_searches
+        @trending_searches ||= begin
+          Workarea::Insights::TrendingSearches.current.results.map do |v|
+            v['query_id']
```

Comment:
The `query_id` helps group similar queries (e.g. `shirts` and `shirt`), so we'll want to use the full query string to avoid output like `red_shirt`.

Reply:
Doesn't seem like this is resolved? You'll want to output the `query_string`.

Reply:
@meowsus looks like this is still outputting `query_id`

Reply:
Derf. Missed this.